src/workspace/block_entry.js

```diff
@@ -17813,12 +17866,12 @@ Entry.block = {
             value = Math.round(value);
             value = Math.max(value, 1);
             value = Math.min(value, 179);
-            Entry.hw.setDigitalPortValue(port, value);
+            Entry.hw.setDigitalPortValue(9, value);
             return script.callReturn();
         }
     },
     "dplay_Robot_run": {
-        "color": "#00979D",
+        "color": "#005766",
```

Comment:
네 혹시라도 색상 변경이 문제가된다면 그부분은 빼고 적용해주셔도 됩니다. 감사합니다


네이버 메일앱에서 보냈습니다.-----Original Message-----
보낸사람: "JY Kim"&lt;notifications@github.com&gt;
받는사람: "entrylabs/entryjs"&lt;entryjs@noreply.github.com&gt;
참조: "SungminSeung"&lt;seungsm@naver.com&gt;, "Author"&lt;author@noreply.github.com&gt;
날짜: 2016. 6. 9 오후 6:44:30
제목: Re: [entrylabs/entryjs] modify block (#341)

In src/workspace/block_entry.js:
&gt;              return script.callReturn();&gt;          }&gt;      },&gt;      "dplay_Robot_run": {&gt; -        "color": "#00979D",&gt; +        "color": "#005766",기존에 이런경우가 없었기에 확인차 물어본거였구요.
내부적으로 이야기 한번 해보고 반영 여부 말해드리도록 하겠습니다.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.